src/krati_Shukla/BankDebitCredit.java

```diff
@@ -0,0 +1,91 @@
+/*Create a Banking System which has the following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+Output :
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+Output :
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+
+
+package krati_Shukla;
+
+public class BankDebitCredit {
+	
+	static int Amount;
+	String custName;
+	int creditCounter;
+	int debitCounter;
+	int printBalance;
+	static int totalPrintBalance;
+	static int totalCreditCounter;
+	static int totalDebitCounter;
+	
+	void customerInfo(String n, int amt) {
+		custName = n;
+		Amount = amt;
+	}
+
+	void debit(int dAmount) {
+		while (dAmount<=Amount) {
```

Comment:
Why while loop here?
you should write if condition to check weather balance is sufficient or not.